### PR TITLE
Msonmez hw week2

### PR DESCRIPTION
**Pull request by msnmz**

### CODE REVIEW

File: Week2/questions.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+const queries = [
+  ({ country }) =>
+    `SELECT city.Name as capitalCity FROM city INNER JOIN country ON city.ID = country.Capital WHERE country.Name = '${country}'`,
+  ({ region }) =>
+    `SELECT Language FROM countryLanguage WHERE CountryCode IN (SELECT CountryCode FROM city INNER JOIN country ON country.Capital = city.ID WHERE country.Region = '${region}')`,
+  ({ language }) =>
+    `SELECT COUNT(city.ID) as numberOfCities, Language FROM city INNER JOIN countryLanguage ON city.CountryCode = countryLanguage.CountryCode WHERE countryLanguage.Language = '${language}'`,
+  ({ region, language }) =>
+    `SELECT country.Name as Country FROM country INNER JOIN city ON city.ID = country.Capital WHERE country.Region = '${region}' AND city.CountryCode IN (SELECT CountryCode FROM countryLanguage WHERE Language = '${language}' AND IsOfficial = 'T')`,
```
Comment (roxana-lafuente) [inline Week2/questions.js]: Query OK!

File: Week2/questions.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  () =>
+    'SELECT COUNT(countryLanguage.Language) as numberOfLanguagesSpoken, country.Continent FROM country INNER JOIN city ON country.Capital = city.ID INNER JOIN countryLanguage ON city.countryCode = countryLanguage.countryCode GROUP BY country.Continent;',
```
Comment (roxana-lafuente) [inline Week2/questions.js]: Query OK!

File: Week2/alert.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'password',
+  database: 'new_world',
+});
+
+const executeQuery = promisify(connection.query.bind(connection));
+
+const languageAlertTrigger = `CREATE TRIGGER max_language_number_exceeded_trigger \
```
Comment (roxana-lafuente) [inline Week2/alert.js]: Trigger is good! Great work!!!

**Comment by msnmz** [comment]
Hi @roxana-lafuente,
Thank you for your great comments.
I made changes on the ER-Diagram and added and ER-Diagram.md that explains entities.

File: Week2/ER-Diagram.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Todo App ER-Diagram
```
[the remark this answers is not in the document]
Comment (msnmz) [reply Week2/ER-Diagram.md]: Yes, you are right about the cross table between the TodoList and the TodoItem tables. Thank you for your great comments.

**Pull request closed without merging**